core/src/main/scala/org/locationtech/rasterframes/expressions/transformers/Mask.scala

```diff
@@ -73,6 +73,15 @@ abstract class Mask(val left: Expression, val middle: Expression, val right: Exp
   override protected def nullSafeEval(targetInput: Any, maskInput: Any, maskValueInput: Any): Any = {
     implicit val tileSer = TileUDT.tileSerializer
     val (targetTile, targetCtx) = tileExtractor(targetExp.dataType)(row(targetInput))
+
+    // Which of these is preferable? companion object Seq contains or pattern match on trait?
```

Comment:
@metasim what do you prefer here?

Reply:
Definitely not `contains`... too wordy. You could also do `targetTile.cellType.isInstanceOf[NoNoData]`.